rosdep/base.yaml

```diff
@@ -1811,6 +1811,16 @@ libhdf5-dev:
   debian: [libhdf5-dev]
   fedora: [hdf5-devel]
   ubuntu: [libhdf5-dev]
+libi2c-dev:
+  arch:
+    pacman:
+      packages: [linux-api-headers]
+  debian:
+    apt:
+      packages: [libi2c-dev]
+  ubuntu:
+    apt:
+      packages: [libi2c-dev]
```

Comment:
The `debian` and `ubuntu` entry doesn't need to call out `apt` and `packages` but can directly list the packages: `[libi2c-dev]` (see other entries for comparison).

I assume for `arch` it is the same but @tfoote will know for sure.